annex/src/example7.cpp

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+*	1 event loop
+*	Listener to print the content of the event and the thread id
+*	Publish and subscriber for other types of data: int, my own strct, etc.
+*	Number as the output
+*****************************************************************************/
+
+#include <iostream>
+
+#include <klepsydra/high_performance/event_loop_middleware_provider.h>
+
+#include <thread>
+
+#include <chrono>
+
+# include "int_publisher.h"
+
+# include "position_publisher.h"
+
+void foo()
+{
+    std::this_thread::sleep_for(std::chrono::seconds(1));
+}
+
+int main() {
+	kpsr::high_performance::EventLoopMiddlewareProvider<16> eventloop(nullptr);
+
+	eventloop.start();
+	
+	std::thread t1(foo);
+	std::thread::id t1_id = t1.get_id();
+	std::cout << "t1's id: " << t1_id << '\n';
```

Comment:
Remove this and previous 2 lines.

Reply:
I removed just the line where the foo function was used. I kept t1's id just to make sure that it is the publisher thread id.